src/environments.jl

```diff
@@ -0,0 +1,67 @@
+project_status() = read(steal_stdout(Pkg.status), String)
+
+function steal_stdout(f)
+  old = stdout
+  rd, wr = redirect_stdout()
+  try
+    f()
+  finally
+    redirect_stdout(old)
+    close(wr)
+  end
+  return rd
+end
+
+@static if VERSION < v"1.1"
+
+update_project() = @msg updateProject(false)
+handle(()->false, "allProjects")
+
+else
+
+update_project() = @msg updateProject(project_info())
+
+atinit(update_project)
```

Comment:
at the point of `__init__`, we haven't established a connection, so `update_project()` won't work at that time.

Reply:
I meant the call to `atinit`, not `update_project`.
Usually we're doing this kind of init stuff (this one and the one above) on the Atom side, which might be an option too.

Reply:
ah I see, yeah this is better to be moved into `__init__`

Reply:
I remove `atinit` stuff at all and moved `update_project` into `connected` handler.

> Usually we're doing this kind of init stuff (this one and the one above) on the Atom side, which might be an option too.

yeah, doing this initialization in Atom would be better in terms of configuring this feature. 
sounds like I should modify the code a bit

Reply:
> sounds like I should modify the code a bit

hmm but for this case, what sends a message should be Julia (since we don't know when to update an environment information in Atom side), and we need to scatter around `update_project` as PR does anyway.

So for configuring this feature, what we can do is to turn on/off [Atom side handler](https://github.com/JunoLab/atom-julia-client/blob/42264dec40c95c128fbd2d885d37a83b83738987/lib/runtime/environments.js#L33-L40) while we need to receive messages from Julia anyway.

So in conclusion I think this PR is okay as is.